sdk/storage/azure-storage-file-share/migration_guide.md

```diff
@@ -0,0 +1,1208 @@
+            <td width="111" valign="top">
+                <p align="right">
+                    FileService
+                </p>
+            </td>
+            <td width="163" valign="top">
+                <p align="right">
+                    clear_range
+                </p>
+            </td>
+            <td width="107" valign="top">
+                <p align="right">
+                    ShareFileClient
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <td width="242" valign="top">
+                <p align="right">
+                    list_ranges
+                </p>
+            </td>
+            <td width="111" valign="top">
+                <p align="right">
+                    FileService
+                </p>
+            </td>
+            <td width="163" valign="top">
+                <p align="right">
+                    get_ranges
+                </p>
+            </td>
+            <td width="107" valign="top">
+                <p align="right">
+                    ShareFileClient
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <td width="242" valign="top">
+                <p align="right">
+                    create_permission_for_share
+                </p>
+            </td>
+            <td width="111" valign="top">
+                <p align="right">
+                    FileService
+                </p>
+            </td>
+            <td width="163" valign="top">
+                <p align="right">
+                    create_permission_for_share
+                </p>
+            </td>
+            <td width="107" valign="top">
+                <p align="right">
+                    ShareClient
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <td width="242" valign="top">
+                <p align="right">
+                    get_permission_for_share
+                </p>
+            </td>
+            <td width="111" valign="top">
+                <p align="right">
+                    FileService
+                </p>
+            </td>
+            <td width="163" valign="top">
+                <p align="right">
+                    get_permission_for_share
+                </p>
+            </td>
+            <td width="107" valign="top">
+                <p align="right">
+                    ShareClient
+                </p>
+            </td>
+        </tr>
+    </tbody>
+</table>
+Please note that while some API functionality does not have a 1-1 mapping, the new Track2 File Share Service adds a lot of new functionality and some old functionality may still be possible, but simply presented in a different manner.
+
+## Build Client with Shared Key Credential
+Instantiate client in Version 2.X
+```python
+from azure.storage.file import FileService
+service = FileService("<storage-account-name>", "<account-access-key>", endpoint_suffix="<endpoint_suffix>")
+```
+
+Instantiate client in Version 12.
```

Comment:
I wonder if we show how to instantiate all clients here since previously there was only one. Maybe add a couple of lines like `share_client = service.get_share_client("myshare")` for each type of client? Doesn't show how to build directly but at least shows a way to get other clients.